wrappers/python/simtk/openmm/app/amberprmtopfile.py

```diff
@@ -248,4 +254,16 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
                 force.setEwaldErrorTolerance(ewaldErrorTolerance)
         if removeCMMotion:
             sys.addForce(mm.CMMotionRemover())
+
+        if switchDistance and nonbondedMethod is not ff.NoCutoff:
+            # make sure it's legal
+            if switchDistance >= nonbondedCutoff:
+                raise ValueError('switchDistance is too large compared '
+                                     'to the cutoff!')
+                if abs(switchDistance) != switchDistance:
```

Comment:
Ah, thanks for the pointer.  I'll clean up that code a bit (it was some of the first OpenMM code I wrote... :)